packages/connector-alipay-native/src/index.test.ts

```diff
@@ -15,18 +19,30 @@ describe('validateConfig', () => {
     jest.clearAllMocks();
   });
 
+  /**
+   * Assertion functions always need explicit annotations.
+   * See https://github.com/microsoft/TypeScript/issues/36931#issuecomment-589753014
+   */
+
   it('should pass on valid config', async () => {
-    await expect(
-      alipayNativeMethods.validateConfig(mockedAlipayNativeConfig)
-    ).resolves.not.toThrow();
+    const validator: ValidateConfig = alipayNativeMethods.validateConfig;
```

Comment:
the type of `validator` looks not right, maybe:
```suggestion
    const validator: typeof alipayNativeMethods.validateConfig = alipayNativeMethods.validateConfig;
```

Reply:
not blocking, add an issue and do it later

Reply:
[LOG-3661](https://linear.app/silverhand/issue/LOG-3661/connector-uts-validator-type-correction) to track this.
Resolved this to unblock upcoming changes.